src/config/data-source.ts

```diff
@@ -0,0 +1,17 @@
+import { DataSource } from "typeorm";
+import dotenv from "dotenv";
+dotenv.config();
+
+export const AppDataSource = new DataSource({
+  type: "mysql",
+  host: process.env.DB_HOST!,
+  port: parseInt(process.env.DB_PORT!),
+  username: process.env.DB_USERNAME!,
+  password: process.env.DB_PASSWORD!,
+  database: process.env.DB_DATABASE!,
+  logging: false,
+  synchronize: true,
```

Comment:
```suggestion
  synchronize: false,
```
E đổi option này thành false và thực hiện config để có thể migrate thủ công nha